pyvista/core/cell.py

```diff
@@ -0,0 +1,104 @@
+"""Define types of cells."""
+from enum import IntEnum
+
+from pyvista import _vtk
+
+
+class CellType(IntEnum):
+    """Define types of cells.
+
+    Warnings
+    --------
+    TODO Add version warning of some cell types.
```

Comment:
@MatthewFlamm It does not seem possible to output VTKVersionWarning with IntEnum. I would add a docstring.

Reply:
```suggestion
    Warnings
    --------
    The following type needs vtk >=v9.0.0 (See `Implementation of vtkTriQuadraticPyramid cell
<https://gitlab.kitware.com/vtk/vtk/-/merge_requests/8295>`_ and `Add Bezier cell types <https://gitlab.kitware.com/vtk/vtk/-/merge_requests/6055>`_).
    * TRIQUADRATIC_PYRAMID
    * BEZIER_TRIANGLE
    * BEZIER_QUADRILATERAL
    * BEZIER_TETRAHEDRON
    * BEZIER_HEXAHEDRON
    * BEZIER_WEDGE
    * BEZIER_PYRAMID
```